test/parallel/test-dns-regress-7070.js

```diff
@@ -4,5 +4,7 @@ const assert = require('assert');
 const dns = require('dns');
 
 // Should not raise assertion error. Issue #7070
-assert.throws(() => dns.resolveNs([])); // bad name
-assert.throws(() => dns.resolveNs('')); // bad callback
+assert.throws(() => dns.resolveNs([]), // bad name
+              /^Error: "name" argument must be a string$/); 
+assert.throws(() => dns.resolveNs(''), // bad callback
+              /^Error: "callback" argument must be a function$/);
```

Comment:
yeah. just added it. thanks for pointing it out. i was wondering about it after my last commit.